src/core/raster/qgsrasterblock.h

```diff
@@ -196,6 +196,20 @@ class CORE_EXPORT QgsRasterBlock
      *  \returns value */
     double value( qgssize index ) const;
 
+    /**
+     * Gives direct access to the raster block data.
+     * The data type of the block must be Qgis::Byte otherwise it returns null pointer.
+     * Useful for most efficient read access.
+     * \note not available in Python bindings
+     * \since QGIS 3.4
+     */
+    const quint8 *byteData() const SIP_SKIP
```

Comment:
Candidate for `inline`? Even though most compiler will happily ignore this and do what's right I think it's the correctthing to do here.

Reply:
My understanding is that inline is totally ignored by modern compilers.

Reply:
Ok, point taken. Merge ahead.